readme.md

```diff
@@ -35,6 +35,16 @@ tempy.directory({prefix: 'name'});
 
 Get a temporary file path you can write to.
 
+### tempy.file.task(callback, options?)
```

Comment:
This does not document that it returns a promise, which I think is important.

Reply:
Done!